go.mod

```diff
@@ -23,7 +23,9 @@ require (
 	google.golang.org/grpc v1.26.0
 	k8s.io/api v0.18.8
 	k8s.io/apimachinery v0.18.8
+	k8s.io/cli-runtime v0.18.8 // indirect
 	k8s.io/client-go v0.18.8
+	sigs.k8s.io/cli-utils v0.19.2
```

Comment:
This should match all the other Kubernetes packages `sigs.k8s.io/cli-utils v0.18.8`